src/vs/workbench/contrib/notebook/browser/contrib/coreActions.ts

```diff
@@ -1093,6 +1093,35 @@ registerAction2(class extends Action2 {
 	}
 });
 
+registerAction2(class extends Action2 {
+	constructor() {
+		super({
+			id: FOCUS_OUTPUT_COMMAND_ID,
+			title: localize('focusOutput', 'Focus output'),
+			category: NOTEBOOK_ACTIONS_CATEGORY,
+			keybinding: {
+				when: ContextKeyExpr.and(NOTEBOOK_EDITOR_FOCUSED, ContextKeyExpr.has(InputFocusedContextKey), EditorContextKeys.editorTextFocus),
```

Comment:
`NOTEBOOK_EDITOR_FOCUSED` is enough I think.

Reply:
It was enough.